stix_shifter_utils/stix_translation/src/utils/transformers.py

```diff
@@ -4,7 +4,7 @@
 import socket
 import re
 import os
-
+import ntpath
```

Comment:
according to https://docs.python.org/3/library/os.path.html ntpath is a Windows specific module. I feel like you have to use more general approach.

Reply:
I know. It works with most UNIX and all windows file path conventions. I couldn't find any more generic approach than this. The main reason I went with this approach is, it works with the sample logs where it is currently used.